packages/react/src/useProviderLink.ts

```diff
@@ -29,13 +29,25 @@ import { NhostReactContext } from './provider'
  * ```
  */
 export const useProviderLink = (options?: ProviderOptions) => {
+  /**
+   * @internal When using Nextjs or any SSR framework, nhost.auth.client.clientUrl will be set to `undefined`
+   * as its value is set to window.location.origin.
+   * This is because the request context is not available when setting up the client `new NhostClient()` outside of
+   * the React/Nextjs context.
+   */
+  const [isSSR, setIsSSR] = useState(true)
```

Comment:
Shouldn't SSR related logic be part of `@nhost/nextjs` instead?

By having SSR related features in `@nhost/react` makes me think if we really need a dedicated separate package for Next.js or just have an `@nhost/react` package that takes care about these kind of things.

Reply:
> Shouldn't SSR related logic be part of `@nhost/nextjs` instead?

maybe, although other React-based frameworks support ssr.

> By having SSR related features in `@nhost/react` makes me think if we really need a dedicated separate package for Next.js or just have an `@nhost/react` package that takes care about these kind of things.

I think there's a separate project to refactor `@nhost/nextjs` with its possible specifics. For instance to be able to get the client url from either the request context on the server-side or from window.location on the client side